recipes/master.rb

```diff
@@ -20,6 +20,15 @@
   action :enable
 end
 
+user node['salt']['master']['api']['user']['name'] do
```

Comment:
This should be left up to a wrapper cookbook.  People may want to use existing users and adding it here would cause resource cloning (which will break in April)